src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -3594,4 +3644,191 @@ make_NetworkOPs (NetworkOPs::clock_type& clock, bool standalone,
+            EXPECT_EQ(txs[ix].second->getLgrSeq(), ledger_index); \
+            EXPECT_EQ(txs[ix].second->getIndex(), txn_index);
+
+        #define EXPECT_TOKEN(ledger_index, txn_index) \
+            EXPECT_EQ(token["ledger"].asInt(),  ledger_index); \
+            EXPECT_EQ(token["seq"].asInt(),     txn_index);
+
+        /*
+        Transaction.db for account contains transactions with ledger/seq
+
+         3|5
+         4|4
+         4|10
+         5|4
+         5|7
+         6|1
+         6|5
+         6|6
+         6|7
+         6|8
+         6|9
+         6|10
+         6|11
+
+        */
+
+        EXPECT_EQ(next(),  1);
+        EXPECT_TX(0, 3, 5);
+        EXPECT_TOKEN(4, 4);
+
+        EXPECT_EQ(next(),  1);
+        EXPECT_TX(0, 4, 4);
+        EXPECT_TOKEN(4, 10);
+
+        EXPECT_EQ(next(),  1);
+        EXPECT_TX(0, 4, 10);
+        EXPECT_TOKEN(5, 4);
+
+        limit = 3;
+
+        EXPECT_EQ(next(),  3);
+        EXPECT_TX(0, 5, 4);
+        EXPECT_TX(1, 5, 7);
+        EXPECT_TX(2, 6, 1);
+        EXPECT_TOKEN(6, 5);
+
+        EXPECT_EQ(next(),  3);
+        EXPECT_TX(0, 6, 5);
+        EXPECT_TX(1, 6, 6);
+        EXPECT_TX(2, 6, 7);
+        EXPECT_TOKEN(6, 8);
+
+        EXPECT_EQ(next(),  3);
+        EXPECT_TX(0, 6, 8);
+        EXPECT_TX(1, 6, 9);
+        EXPECT_TX(2, 6, 10);
+
+        EXPECT_TOKEN(6, 11);
+        EXPECT_EQ(next(),  1);
+        EXPECT_TX(0, 6, 11);
+        EXPECT(token["ledger"].isNull());
+        EXPECT(token["seq"].isNull());
+
+        /*
+        sqlite> select LedgerSeq, TxnSeq from AccountTransactions where
+                Account = 'rfu6L5p3azwPzQZsbTafuVk884N9YoKvVG'
+                ORDER BY AccountTransactions.LedgerSeq DESC,
+                AccountTransactions.TxnSeq DESC;
+        6|11
+        6|10
+        6|9
+        6|8
+        6|7
+        6|6
+        6|5
+        6|1
+        5|7
+        5|4
+        4|10
+        4|4
+        3|5
+        */
+
+        token = Json::nullValue;
+        forward = false;
+        limit = 2;
+
+        EXPECT_EQ(next(),  2);
+        EXPECT_TX(0, 6, 11);
+        EXPECT_TX(1, 6, 10);
+        EXPECT_TOKEN(6, 9);
+
+        EXPECT_EQ(next(),  2);
+        EXPECT_TX(0, 6, 9);
+        EXPECT_TX(1, 6, 8);
+        EXPECT_TOKEN(6, 7);
+
+        limit = 3;
+        EXPECT_EQ(next(),  3);
+        EXPECT_TX(0, 6, 7);
+        EXPECT_TX(1, 6, 6);
+        EXPECT_TX(2, 6, 5);
+        EXPECT_TOKEN(6, 1);
+
+        EXPECT_EQ(next(),  3);
+        EXPECT_TX(0, 6, 1);
+        EXPECT_TX(1, 5, 7);
+        EXPECT_TX(2, 5, 4);
+        EXPECT_TOKEN(4, 10);
+
+        EXPECT_EQ(next(),  3);
+        EXPECT_TX(0, 4, 10);
+        EXPECT_TX(1, 4, 4);
+        EXPECT_TX(2, 3, 5);
+
+        EXPECT(token["ledger"].isNull());
+        EXPECT(token["seq"].isNull());
+
+        #undef EXPECT_TX
+        #undef EXPECT_TOKEN
+
+    }
+};
+
+BEAST_DEFINE_TESTSUITE_MANUAL(AccountTxPager,ripple_app,ripple);
```

Comment:
This is defined as manual, cause it needs to find the fixtures directory somehow, which currently just looks for TEST_FIXTURES env var.  

Ideas?